src/main/java/com/ollymonger/dynmap/portals/DynmapPortalsPlugin.java

```diff
@@ -34,6 +36,22 @@ public void onEnable() {
     @EventHandler
     public void onPortalCreate(PortalCreateEvent event) {
         getLogger().info("Portal created");
+        List<BlockState> blocks = event.getBlocks();
+
+        for (int i = 0; i < blocks.size(); i++) {
+            Block block = blocks.get(i).getBlock();
+            World world = block.getWorld();
```

Comment:
You don't use `world`, `x`, `y`, or `z` until after the if statement - so move them below